src/cls/rbd/cls_rbd.cc

```diff
@@ -4651,6 +4651,13 @@ int uuid_get(cls_method_context_t hctx, std::string *mirror_uuid) {
   return 0;
 }
 
+void sanitize_entity_inst(entity_inst_t* entity_inst) {
+  // make all addrs of type ANY because the type isn't what uniquely
+  // identifies them and clients and on-disk formats can be encoded
+  // with different backwards compatibility settings.
+  entity_inst->addr.set_type(entity_addr_t::TYPE_ANY);
```

Comment:
@dillaman Just wondering about backporting to mimic and luminous. `TYPE_ANY` was introduced in nautilus...

Reply:
Even better ... I'll tweak the backport flag to nautilus only.